src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardSyncStrategyType.java

```diff
@@ -0,0 +1,11 @@
+package com.amazonaws.services.kinesis.clientlibrary.lib.worker;
+
+/**
+ * Types of ShardSyncStrategy} implemented in KCL.
+ */
+public enum ShardSyncStrategyType {
+
+        PERIODIC, /* Shard sync are performed periodically */
```

Comment:
nit: Could the Javadoc be on the line above the value.

Reply:
Changed.